.github/pull_request_template.md

```
@@ -0,0 +1,5 @@
+## Choose the appropriate template for your PR:
+
+- [Feature PR](./PULL_REQUEST_TEMPLATE/feature_template.md)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)